### DISCUSSION TITLE
Hide table records if view policy is false.

### DISCUSSION THREAD

**Discussion by HenrijsS** [filamentphp/filament#13579] [Help]
### Package

Table builder

### Package Version

v3.2.93

### How can we help you?

The [Listing Records](https://filamentphp.com/docs/3.x/panels/resources/listing-records#authorization) says that if viewAny is true, then the users can see ALL records, even though "View" is false.

That means they can see all information about the most important fields, even though they don't have access to it.

I want to only show table records that have `view = true` policy.

**Comment by leandrocfe** [comment]
Actually, you should use `modifyQueryUsing` or Laravel scope to filter records by user
https://filamentphp.com/docs/3.x/panels/resources/listing-records#customizing-the-table-eloquent-query
